setup.py

```diff
@@ -1,37 +0,0 @@
-from pathlib import Path
-
-from setuptools import find_packages, setup
-
-parent_dir = Path(__file__).resolve().parent
-
-setup(
-    name="up42-py",
-    version=parent_dir.joinpath("up42/_version.txt").read_text(encoding="utf-8"),
-    author="UP42",
-    author_email="support@up42.com",
-    description="Python SDK for UP42",
-    long_description=parent_dir.joinpath("README.md").read_text(),
-    long_description_content_type="text/markdown",
-    url="https://github.com/up42/up42-py",
-    license="MIT",
```

Comment:
We need to change pyproject, otherwise it doesn't have a static license which is problematic since we can change the licensing affecting old versions